libsolidity/ast/ASTJsonConverter.cpp

```diff
@@ -833,6 +833,18 @@ bool ASTJsonConverter::visit(Literal const& _node)
 	return false;
 }
 
+bool ASTJsonConverter::visit(FormalDocumentation const& _node)
+{
+	Json::Value text{*_node.text()};
+	std::vector<pair<string, Json::Value>> attributes = {
+		make_pair("text", text),
+		make_pair("format", "NatSpec")
```

Comment:
Why specify this?

Reply:
I had the impression that it would be better to be able to switch the formal / structured doc format. At least under the assumption that NatSpec is not accepted thoughout the community. It's not needed right now, but I wanted to take the chance within this change to support it. Do you think it's not necessary and could be added later?

Reply:
Especially since the ast json output is solidity version specific, I don't think we need the ability to switch.